shared/actions/wallets.tsx

```diff
@@ -13,7 +13,7 @@ import * as RouteTreeGen from './route-tree-gen'
 import * as Flow from '../util/flow'
 import * as Router2Constants from '../constants/router2'
 import HiddenString from '../util/hidden-string'
-import logger from '../logger'
+import _logger from '../logger'
```

Comment:
why?

Reply:
this is because i have eslint warning us on variable shadowing. honestly most cases of _logger should likely go away and use the better saga logger instead